### ISSUE TITLE
Printing a file always results in "undesirable" output

### ISSUE THREAD

**Issue by parkdweller** [retext-project/retext#408]
Hi, I'm trying to print a list made in retext markdown (I'm using version 6.0.2 installed from the Debian stretch repos, but looking at the changelog, there's no mention of this issue for version 7).

When I tried to print in my laser printer, the document came out blank. Thinking there was a problem with the printer, I tried "printing" the same document as a pdf, and what I found was that the document was being printed, but with letters so light-grey that they were almost invisible:
![1](https://user-images.githubusercontent.com/15857923/48505894-6d123800-e848-11e8-882e-1163645285a1.png)

Switching to webkit rendering, I tried printing it as a pdf again, and the output seems to use the same theme I'm using on the desktop (white characters on a dark background), which is very wasteful when printed on paper:
![2](https://user-images.githubusercontent.com/15857923/48505895-6d123800-e848-11e8-95f2-a601aace3c0e.png)

In other words, I can't manage to print my documents in the standard of black letters on white background. Am I missing something, or is this a bug?

**Comment by mitya57** [comment]
Thanks for your bug report.

Do you use any custom CSS or QSS?

Can you please attach your configuration file (`~/.config/ReText project/ReText.conf`)?

**Comment by parkdweller** [comment]
Hi @mitya57 , thanks for the reply.

I don't use any custom CSS or QSS (you're talking about the settings option to import a stylesheet file, right?)

One more thing I found out: exporting the documents as .pdf or .odt from the "export" menu option gives the same results as the ones I described before. Exporting as html, on the other hand, gives the "correct" output: black letters on white background. 

Here's my ReText.conf (I removed the recent documents because I use too much profanity in the titles):

[General]
autoSave=true
livePreviewByDefault=true
recentFileList=REDACTED
saveWindowGeometry=true
windowGeometry=@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\0\0\0\x87\0\0\0\x36\0\0\x5\xc2\0
0\x3\xb8\0\0\0\x8d\0\0\0[\0\0\x5\xbc\0\0\x3\xb2\0\0\0\0\0\0\0\0\a\x80)

**Comment by Gawarmel** [comment]
Can verify the same problem. Installed retext on Kubuntu 18.04 with apt install retext. pdf and odt export lead to super light grey output. tried then the pip3 install. Same result. Temporarily switched back to older version of retext that used to work for years.    ;-)

**Comment by qoheniac** [comment]
Same probleme here using the vanilla ReText my distro (Fedora 30) provides.

ReText 7.0.3
PyMarkups 2.0.0
Qt 5.12.4
GNOME 3.32.2 with Adwaita-dark theme

**Comment by mitya57** [comment]
I have pushed a commit which should fix issues with printing and PDF export. Please test it if you can.

To test, you can just clone the repository from GitHub and start `./retext.py` from there.

**Comment by qoheniac** [comment]
Printing and PDF export now indeed work fine but now "Preview" shows nothing.

**Comment by mitya57** [comment]
@qoheniac Most probably you have Python-Markdown 2.x while the latest ReText master supports only ≥ 3.0.

Please update your Python-Markdown or revert commit 59dcfc6ef762f756d070981287dc41d22b81e248 in ReText.

**Comment by qoheniac** [comment]
I updated Markdown and Markups and now it works fine. Thanks! :)
